lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss

```diff
@@ -130,3 +130,81 @@
   }
 
 }
+
+.campaign--pitch {
+  .cta.-persistent {
+    display: none;
+  }
+
+  &.-persistent-beta {
```

Comment:
Can you switch this to `.optimizely-*` naming convention we've used for other A/B test classes?